src/codecs/resize/resize.ts

```diff
@@ -1,21 +1,28 @@
-import { nativeResize, NativeResizeMethod } from '../../lib/util';
+import { nativeResize, NativeResizeMethod, drawableToImageData } from '../../lib/util';
 
-export function resize(data: ImageData, opts: ResizeOptions): ImageData {
+function getCoverOffsets(sw: number, sh: number, dw: number, dh: number) {
+  const currentAspect = sw / sh;
+  const endAspect = dw / dh;
+
+  if (endAspect > currentAspect) {
+    const newSh = dh / (dw / sw);
```

Comment:
I think with some shuffling, we can make the math a bit more “obvious”:
`dh / (dw/sw) = sw * dh / dw = sw/endAspect`

WDYT?

Reply:
I'm not sure what's going on there, but it's definitely an invalid left-hand assignment. What am I missing?

Reply:
Ohhh you mean change to `sw / endAspect`! I see now

Reply:
Lol sorry. I am suggesting to use the expression on the very right and showing how I got there. This is not supposed to be JavaScript, just math :D My bad.

Reply:
... GitHub really needs to fix their stupid code-review UI to update when someone comments.